### PR TITLE
chore: synced another batch of deps

### PR DESCRIPTION
**Pull request by viliusl**

### CODE REVIEW

**Comment by viliusl** [comment]
@yanivefraim @roymiloh please:)

File: examples/client-react/package.json [hunk 1]
```diff
@@ -23,10 +23,10 @@
     "yoshi": "~1.1.0"
   },
   "dependencies": {
-    "babel-polyfill": "^6.23.0",
+    "babel-polyfill": "~6.23.0",
     "react": "~15.6.1",
     "react-dom": "~15.6.1",
-    "react-i18next": "^1.10.1"
+    "react-i18next": "~4.6.4"
```
Comment (roymiloh) [inline examples/client-react/package.json]: Such bump:) few peer deps [are missing](https://github.com/i18next/react-i18next/blob/master/package.json#L58), you decide if we remain with v1 or upgrade to v4, but please fix it if you decide to upgrade to v4.

Comment (viliusl) [inline examples/client-react/package.json]: @roymiloh there is more interesting problem - this dependency is actually unused. Which is also verified by passing ci test.

Unused dependencies is smth else that I will want to attack, but it's out of scope.

I will remove it altogether then.

**Comment by viliusl** [comment]
@roymiloh - better?:)

**Pull request merged**